src/components/concept-page/concept-page.styles.scss

```diff
@@ -0,0 +1,67 @@
+.page-break {
+    // height: 180px;
+    height: 40vh;
+    width:100%;
+    color: white;
+}
+
+.page-container {
+    // height: 960px;
+    // height: 160vh;
+    height: 120vh;
+
+    width: 100vw;
+}
+
+.title {
+    font-family: 'Times New Roman', Times, serif;
+    font-size: 3.75vw;
+    color: #5A4537;
+    letter-spacing: 4.32px;
+    display: flex;
+    justify-content: center;
+}
+
+.subheading {
+    font-family: YuMincho;
+    font-size: 2vw;
+    color: #5A4537;
+    padding-top: 13.5vh;
+    width: 36vw;
+    padding-left: 18vw;
+
+}
+
+.two-halfs-container {
+    display: flex;  
+    padding-top: 12vh;
+}
+
+.left-half-container {
+    width: 50vw;
+}
+
+.textbox {
+    font-family: YuGothic;
+    font-size: 1.25vw;
```

Comment:
shouldn't use vw/vh for text here.
when its fancy text in the background it might be ok, but when its actual content, it has the possibility of going way too small for users to read.
use rem/em. refer to that vid I sent you.

Reply:
Changed to em